File: Python/update_contact.py
## This script requires "requests": http://docs.python-requests.org/
## To install: pip install requests

import requests
import json

api_key = "YOUR_API_KEY"
domain = "YOUR_DOMAIN"
password = "x"

# Id of the contact to be updated
contact_id = 'CONTACT_ID'

contact_info = { "priority" : "Super Hero" }
headers = { "Content-Type" : "application/json" }

r = requests.put("https://"+ domain +".freshdesk.com/api/v2/contacts/"+contact_id, auth = (api_key, password), data = json.dumps(contact_info), headers = headers)

if r.status_code == 200:
  print "Contact updated successfully, the response is given below" + r.content
else:
  print "Failed to update contact, errors are displayed below,"
  response = json.loads(r.content)
  print response["errors"]

  print "x-request-id : " + r.headers['x-request-id']
  print "Status Code : " + r.status_code

File: PHP/update_contact.php
<?php 

$api_key = "API_KEY";
$password = "x";
$yourdomain = "YOUR_DOMAIN";

$contact_data = json_encode(array(
  "job_title" => "Super user",
  "name" => "New name"
));

// Id of the contact to be updated
$contact_id = 6005608974;

$url = "https://$yourdomain.freshdesk.com/api/v2/contacts/$contact_id";

$ch = curl_init($url);

$header[] = "Content-type: application/json";
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'PUT');
curl_setopt($ch, CURLOPT_HTTPHEADER, $header);
curl_setopt($ch, CURLOPT_HEADER, true);
curl_setopt($ch, CURLOPT_USERPWD, "$api_key:$password");
curl_setopt($ch, CURLOPT_POSTFIELDS, $contact_data);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);

$server_output = curl_exec($ch);
$info = curl_getinfo($ch);
$header_size = curl_getinfo($ch, CURLINFO_HEADER_SIZE);
$headers = substr($server_output, 0, $header_size);
$response = substr($server_output, $header_size);

if($info['http_code'] == 200) {
  echo "Contact updated successfully, the response is given below \n";
  echo "Response Headers are \n";
  echo $headers."\n";
  echo "Response Body \n";
  echo "$response \n";
} else {
  if($info['http_code'] == 404) {
    echo "Error, Please check the end point \n";
  } else {
    echo "Error, HTTP Status Code : " . $info['http_code'] . "\n";
    echo "Headers are ".$headers;
    echo "Response are ".$response;
  }
}

curl_close($ch);

?>